### DISCUSSION TITLE
Video stream

### DISCUSSION THREAD

**Discussion by veshapidze** [mikey0000/Mammotion-HA#68] [Ideas]
Is it currently possible to pull the video stream in order to integrate it into an NVR? It would be nice for reviewing obstacles, how the robot gets hung up, or for security reasons.

**Comment by R0M107** [comment]
No it's not currently possible on the HA Integration

**Reply by veshapidze** [reply]
Are there plans to implement it?

**Reply by R0M107** [reply]
Come on the discord server, you can directly ask the devs :)
And it's planned but I don't think it will happen soon.

**Reply by mikey0000** [reply]
It will happen when it happens 😊

**Reply by veshapidze** [reply]
Could you post the Discord link? Thanks

**Reply by mikey0000** [reply]
It's on the readme of this project.

**Reply by MacValley** [reply]
I also would love to see the video stream! Unfortunately I cannot access Discord, keeps telling me invalid site when I want to verify via email, aargh.

**Reply by mikey0000** [reply]
Coming soon....

**Reply by JoshFDXGUS** [reply]
Hello - do you have a timeline for the camera stream?

**Reply by mikey0000** [reply]
Done, jump on discord, need a custom card.

**Reply by JoshFDXGUS** [reply]
where do i need to go on Discord?

**Reply by mikey0000** [reply]
https://discord.com/channels/1247286396297678879/1368820047354662962

**Reply by JoshFDXGUS** [reply]
> https://discord.com/channels/1247286396297678879/1368820047354662962

this takes me to a blank page on discord.

![image](https://github.com/user-attachments/assets/8e1c86d3-d70b-47ae-89f7-6cf8e6cfd6c3)

**Reply by mikey0000** [reply]
Did you join the mammotion discord? Linked on the readme

**Reply by JoshFDXGUS** [reply]
got the camera card created however keep getting Connection Timeout.  Click to retry.  Native Mammotion app loads camera no issues

![image](https://github.com/user-attachments/assets/f97cf89a-611a-4c12-a404-e7f57e845e17)

![image](https://github.com/user-attachments/assets/4e460fe0-92d9-4f42-a9ec-ab46ea78c6e3)

**Reply by Null-Mind** [reply]
> got the camera card created however keep getting Connection Timeout. Click to retry. Native Mammotion app loads camera no issues
> 
> ![image](https://private-user-images.githubusercontent.com/151108965/444284546-f97cf89a-611a-4c12-a404-e7f57e845e17.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDc4MjEwMDgsIm5iZiI6MTc0NzgyMDcwOCwicGF0aCI6Ii8xNTExMDg5NjUvNDQ0Mjg0NTQ2LWY5N2NmODlhLTYxMWEtNGMxMi1hNDA0LWU3ZjU3ZTg0NWUxNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDUyMVQwOTQ1MDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZWMwOTUxMWE5YjFjMTE1MDJjNDQ1YzVkNDIxM2M4MGYzYmI5ZjBhZTc2ZGIyNzczMjhhZWZiZDI0M2E3MjY1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.onkPcH5bVBA4k0Mkkqyh8OH412m25gVVGInMPtOUz1U)
> 
> ![image](https://private-user-images.githubusercontent.com/151108965/444282435-4e460fe0-92d9-4f42-a9ec-ab46ea78c6e3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDc4MjEwMDgsIm5iZiI6MTc0NzgyMDcwOCwicGF0aCI6Ii8xNTExMDg5NjUvNDQ0MjgyNDM1LTRlNDYwZmUwLTkyZDktNGY0Mi1hOWVjLWFiNDZlYTc4YzZlMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDUyMVQwOTQ1MDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04Y2MwNWE1Y2I4YWFkZTQ1M2U4ZGUwZGU4OGUyZDM2YjdiMDk0OGYxYTI0Njc3N2NmOWI0YTM5ZTg1MjM5YzJhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.6Ov55lOpZ3GnZmVS_NJ1wI7ZbxJDiFmm0MlYfm52urU)

Hi, any chance you can share those cards ? how did you got a map ?

**Comment by Symbiot78** [comment]
I tried the advanced camera card and added the sensor. I selected the provider as automatic.
that gives me the camera stream from the yuka. 
I don't know what to use it for.. but it's there. 

but I think - if you continuously stream live view your battery will pay dearly.

this works.. it takes 4-6 seconds before the stream appears:


type: custom:advanced-camera-card
cameras:
  - camera_entity: camera.yuka_mini_none
    live_provider: auto
